linkis-spring-cloud-services/linkis-service-gateway/linkis-gateway-server-support/src/main/scala/org/apache/linkis/gateway/ujes/parser/EntranceRequestGatewayParser.scala

```diff
@@ -50,13 +63,57 @@ class EntranceRequestGatewayParser extends AbstractGatewayParser {
           } else {
             ServiceInstance(GatewayConfiguration.ENTRANCE_SPRING_NAME.getValue, null)
           }
-        } else {
+        } else if (execId.startsWith(ZuulEntranceUtils.EXEC_ID)) {
+          // parse by execId
           ZuulEntranceUtils.parseServiceInstanceByExecID(execId)(0)
+        } else {
+          // build JobInstance by taskId
```

Comment:
Whether it is necessary to add the switch ENTRANCE_FAILOVER_ENABLED to judge

Reply:
No, this is not directly related to failover, the logic here is just to support taskId